pkg/maps/ctmap/ctmap.go

```diff
@@ -202,10 +202,6 @@ func InitMapInfo(tcpMaxEntries, anyMaxEntries int, v4, v6 bool) {
 		anyMaxEntries, natMaps[mapTypeIPv6AnyGlobal])
 }
 
-func init() {
-	InitMapInfo(option.CTMapEntriesGlobalTCPDefault, option.CTMapEntriesGlobalAnyDefault, true, true)
-}
-
```

Comment:
Why is this being removed?

Reply:
Because it ends up creating map objects with the wrong info. At `init` point, options such as `EnableNodePort` may not have their final information. We have a call early on startup to replace this.